cunumeric/linalg/cholesky.py

```diff
@@ -99,7 +139,7 @@ def gemm(context, p_output, k, i, lo, hi):
 
 
 # TODO: We need a better cost model
-def choose_color_shape(runtime, shape):
+def choose_color_shape(runtime: Runtime, shape: Shape) -> tuple[int, int]:
```

Comment:
I think we should change the body of this function to make the return type be `Shape`.